src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -257,250 +211,305 @@ class BasePaymentsPage extends React.Component {
                                 onPress={() => {
-                                    this.setState({
-                                        showConfirmDeleteContent: true,
+                                    setShouldShowDefaultDeleteMenu(false);
+
+                                    // Wait for the previous modal to close, before opening a new one. A modal will be considered completely closed when closing animation is finished.
+                                    // InteractionManager fires after the currently running animation is completed.
+                                    // https://github.com/Expensify/App/issues/7768#issuecomment-1044879541
+                                    InteractionManager.runAfterInteractions(() => {
+                                        if (Permissions.canUsePasswordlessLogins(props.betas)) {
+                                            makeDefaultPaymentMethod();
+                                        } else {
+                                            setShowPassword({
+                                                shouldShowPasswordPrompt: true,
+                                                passwordButtonText: translate('paymentsPage.setDefaultConfirmation'),
+                                            });
+                                        }
                                     });
                                 }}
-                                style={[shouldShowMakeDefaultButton ? styles.mt4 : {}]}
-                                text={this.props.translate('common.delete')}
-                                danger
+                                text={translate('paymentsPage.setDefaultConfirmation')}
                             />
-                        </View>
-                    ) : (
-                        <ConfirmContent
-                            onConfirm={() => {
-                                this.hideDefaultDeleteMenu(false);
-                                this.deletePaymentMethod();
-                            }}
-                            onCancel={this.hideDefaultDeleteMenu}
-                            contentStyles={!this.props.isSmallScreenWidth ? [styles.sidebarPopover, styles.willChangeTransform] : undefined}
-                            title={this.props.translate('paymentsPage.deleteAccount')}
-                            prompt={this.props.translate('paymentsPage.deleteConfirmation')}
-                            confirmText={this.props.translate('common.delete')}
-                            cancelText={this.props.translate('common.cancel')}
-                            anchorPosition={{
-                                top: this.state.anchorPositionTop,
-                                right: this.state.anchorPositionRight,
+                        )}
+                        {isPayPalMeSelected && (
+                            <Button
+                                onPress={navigateToAddPaypalRoute()}
```

Comment:
@multijump Here is the problem, it should be

`onPress={navigateToAddPaypalRoute}`

Reply:
Please create a follow-up PR soon, thanks!

Reply:
Hi, @Santhosh-Sellavel 
Sorry for issue.
I just created followup PR
https://github.com/Expensify/App/pull/23816
But I couldn't record screen because paypal button didn't appear in my dev env.
Can you please let me know how can I add paypal button?
I will test and record screen.
Thanks.